tests/playwright-test/test-server-connection.spec.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { test as baseTest, expect } from './ui-mode-fixtures';
+
+import { TestServerConnection } from '../../packages/playwright/src/isomorphic/testServerConnection';
```

Comment:
You probably want to import from `lib`, not `src`, otherwise path mapping goes crazy.

Reply:
good idea! that's probably what tripped up the linter.